server/v2/cometbft/client/grpc/reflection/reflection.go

```diff
@@ -0,0 +1,45 @@
+package reflection
```

Comment:
this doesnt need to be part of cometbft, users would want this with any consensus. discussed on the arch call to move this to a module somehow